CoinDock-api/database/migrations/2022_07_08_055255_update_coins_table_columns.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('coins', function (Blueprint $table) {
+            $table->string('coin_id');
+            $table->integer('is_crypto');
+            $table->integer('status')->default(0);
+            $table->string('img_path')->default('');
```

Comment:
use nullable instead of using an empty string as the default value.

Reply:
changed